webview/src/plots/components/ErrorsModal.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styles from './styles.module.scss'
+import { Error } from '../../shared/components/icons'
+import { PlotsState } from '../store'
+import { useModalOpenClass } from '../hooks/useModalOpenClass'
+
+export const ErrorsModal: React.FC = () => {
+  const errors = useSelector((state: PlotsState) => state.webview.plotErrors)
+  useModalOpenClass()
+
+  return (
+    <div className={styles.errorsModal} data-testid="errors-modal">
+      <h3 className={styles.errorsModalTitle}>
+        <Error className={styles.errorsModalIcon} width="20" height="20" />
+        Errors
+      </h3>
+      <table>
+        <tbody>
+          {errors.map(({ path, revs }, index) => (
+            <React.Fragment key={index}>
+              <tr>
+                <th colSpan={2} className={styles.errorsModalPlot}>
+                  {path}
+                </th>
+              </tr>
+              {revs.map(({ rev, msg }, index) => (
+                <tr key={index}>
```

Comment:
In this context, `index` is not a good  idea for a `key`. 

What happens if you have four errors and the 2nd gets removed? The 3rd and 4th will re-render because they do not have the same key anymore (more than re-rendering is actually happening, but let's keep this simple).  It isn't a big deal because errors are just text, and the list is short, but the longer it gets and the more complex the node is, the bigger of a problem it will be.

Keys do not need to be clean as IDs. As long as they are text and can be compared, it's okay. You could potentially have a block of text, and it wouldn't be a problem (it would be a bad idea, though).

In this case, what makes the error unique is the combination of `rev` and `msg`. These should not change for a specific error.

You can simply do `<tr key={`${rev}-${msg}`}>`.

Reply:
Good catch! Updated the keys.